mypy/checkexpr.py

```diff
@@ -1628,8 +1628,20 @@ def is_valid_var_arg(self, typ: Type) -> bool:
 
     def is_valid_keyword_var_arg(self, typ: Type) -> bool:
         """Is a type valid as a **kwargs argument?"""
-        return is_subtype(typ, self.chk.named_generic_type(
-            'builtins.dict', [self.named_type('builtins.str'), AnyType()]))
+        if self.chk.options.python_version[0] >= 3:
+            return is_subtype(typ, self.chk.named_generic_type(
+                'builtins.dict', [self.named_type('builtins.str'),
+                                  AnyType()]))
+        else:
+            return (
+                is_subtype(typ, self.chk.named_generic_type(
```

Comment:
Unfortunately that didn't work (It broke the the test case with `str`).  I also tried

```py
UnionType.make_union([self.named_type('builtins.unicode'), self.named_type('builtins.str')])
```

and

```py
self.named_type('builtins.basestring')
```